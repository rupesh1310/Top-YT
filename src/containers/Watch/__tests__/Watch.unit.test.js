import React from 'react';
import {shallow} from 'enzyme';
import {Watch} from '../Watch';

describe('Watch', () => {
  test('renders', () => {
    const wrapper = shallow(<Watch/>);
    expect(wrapper).toMatchSnapshot();
  });
});