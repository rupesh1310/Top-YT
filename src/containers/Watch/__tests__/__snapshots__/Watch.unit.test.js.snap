// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Watch renders 1`] = `
<Connect(WatchContent)
  bottomReachedCallback={[Function]}
  videoId={null}
/>
`;
